hazelcast/proxy/reliable_topic.py

```diff
@@ -587,6 +587,8 @@ def publish(self, message: MessageType) -> Future[None]:
             return self._add_or_overwrite(topic_message)
         elif overload_policy == TopicOverloadPolicy.DISCARD_NEWEST:
             return self._add_or_discard(topic_message)
+        else:
+            raise ValueError(f"Unexpected overload policy is passed {overload_policy}")
```

Comment:
Probably we are checking this in config validation but it is to make type checker happy

Reply:
Yes